.github/workflows/build.yml

```diff
@@ -10,10 +10,30 @@ env:
   GITHUB_ORGANIZATION: philips-software
 
 jobs:
-  build-and-test:
-    name: Build maven project
+  build-and-test-frontend:
+    name: Build frontend project
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout 
+        uses: actions/checkout@v2
 
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '15'
```

Comment:
Is Java really required to build and test the Flutter front end?

Reply:
Removed it.. :)